systray_linux.go

```diff
@@ -18,3 +62,268 @@ func (item *MenuItem) SetIcon(iconBytes []byte) {
 // .ico/.jpg/.png for other platforms.
 func (item *MenuItem) SetTemplateIcon(templateIconBytes []byte, regularIconBytes []byte) {
 }
+
+func addOrUpdateMenuItem(item *MenuItem) {
+}
+
+func setInternalLoop(_ bool) {
+	// nothing to action on Linux
+}
+
+func registerSystray() {
+}
+
+func addSeparator(id uint32) {
+}
+
+func hideMenuItem(item *MenuItem) {
+}
+
+func showMenuItem(item *MenuItem) {
+}
+
+func nativeLoop() int {
+	nativeStart()
+	select {
+	case <- quitChan:
+		break
+	}
+	nativeEnd()
+	return 0
+}
+
+func nativeEnd() {
+	systrayExit()
+}
+
+func quit() {
+	close(quitChan)
+}
+
+func nativeStart() {
+	systrayReady()
+
+	intro := introspect.Introspectable(`<node>
```

Comment:
Wouldn't it be possible to use fyne bundle to store the file in git as the xml file directly?

Reply:
Additionally have you looked at https://github.com/amenzhinsky/dbus-codegen-go ? I feel like a codegen dbus would be nicer to use and seems less brittle.

Reply:
Yes, bundling is one possibility for the solution, however as you comment I am not sure that working from XML is the best end solution.
The hint for dbus-codegen looks like just the thing that could tidy this up, thanks.

Reply:
Done :)